lib/quicksilver-config-client.js

```diff
@@ -51,6 +53,21 @@ class QuicksilverConfigClient {
   getExchangeSettings() {
     return true;
   }
+
+  /**
+   * Check that configuration file define in .env as QS_CONFIG_PATH is accessable.
```

Comment:
Environment is not relevant here. We use env vars to configure test instance only. What you are inspecting here is `qsConfigPath`.